onnxruntime/core/platform/windows/env.cc

```diff
@@ -67,7 +67,9 @@ class WindowsThread : public EnvThread {
   // This function is called when the threadpool is cancelled.
   // TODO: Find a way to avoid calling TerminateThread
   void OnCancel() {
+#if WINAPI_FAMILY_PARTITION(WINAPI_PARTITION_DESKTOP)
```

Comment:
In the store context the thread will keep running?

This seems like it could potentially cause problems?

Reply:
Yes, there's no way to TerminateThread in UWP and it will keep running to finish its job. Though this shouldn't be called anyways (you should finish all threads before cancelling the threadpool, and terminating the thread leaks handles/memory/mutexes/whatever the thread was holding).

This will also match ORT's behavior elsewhere (https://github.com/microsoft/onnxruntime/blob/01ca6392cb165f3ebad6fc3c28016adad5a26688/onnxruntime/core/platform/posix/env.cc#L157-L160 does nothing when you cancel a running thread).